src/Repository/index.js

```diff
@@ -61,17 +71,17 @@ class Repository {
         return multipleFilesByPath
     }
 
-    async getHeadRef(defaultBranch) {
+    async getHeadRef(branchName) {
         const result = await this.github.git.getRef({
```

Comment:
Instead of checking 404 above when called.

Can we throw an error here? (BranchAlreadyExists) and catch that